src/main/java/pl/sda/javapoz/model/Info.java

```diff
@@ -1,8 +1,8 @@
 package pl.sda.javapoz.model;
 
-/**
- * Created by RENT on 2017-03-27.
- */
+import javax.persistence.Entity;
+
+@Entity
```

Comment:
To chyba też nie była encja zapisywana do bazy. Z tego co pamiętam to gość wymyslił to do pojawiajacych się komentarzy na stronie. w momencie wypozyczenia. Jesli sie udalo to pojawial sie zielony prostokąt, a jak nie to czerwony. Jakos to z thymeleaf'em połaczone.

Reply:
Wyjasnione